install

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# This script will eventually be replaced with `databricks labs install ucx` command.
```

Comment:
why do we need this file? please remove.

Reply:
Removed